app/assets/stylesheets/active_admin/print.scss

```diff
@@ -27,85 +31,72 @@ a {
   text-decoration: none;
 }
 
-h3 {
-  font-weight: bold;
-  margin-bottom: .5em;
-}
-
-// Header
-#header {
-  float: left;
-
-  #tabs, .tabs, #utility_nav {
-    display: none;
-  }
-
-  h1{
-    font-weight: bold;
-  }
-}
-
-.flashes {
+// Hidden
+.unsupported_browser,
```

Comment:
Avoid using id selectors<br>Avoid qualifying attribute selectors with an element.<br>Selector `unsupported_browser` should be written in lowercase with hyphens<br>Selector `member_link` should be written in lowercase with hyphens<br>Selector `index_footer` should be written in lowercase with hyphens<br>Selector `pagination_information` should be written in lowercase with hyphens<br>Selector `filters_sidebar_section` should be written in lowercase with hyphens